core/src/empty/empty.tsx

```diff
@@ -0,0 +1,30 @@
+import s from "./empty.module.css";
+import { createElement } from "react";
+import { Button } from "../button/button";
+import cloudImg from "./empty1.jpg";
```

Comment:
Please use SVG. If you're having difficult exporting it then please ask @ng-hai or me for help